src/render/blog/business-logic-definition.html.md

```diff
@@ -0,0 +1,96 @@
+---
+title: What does business logic mean for us
+description: We will find an answer what does the business logic mean for us. We would transform user stories to domain objects and then model the solution.
+author: tomaszmichalak
+keywords: business,requirements,user-stories,business-logic,domain-object,domain-space,solution-space
```

Comment:
It doesn't work like the tags unfortunately. And this would be to much, I'd suggest:
```suggestion
keywords: business-logic,domain-driven-design,user-stories
```
According to SEO rules, the good practice says to put up to 3-5 tags.